mnemosyne/cli.py

```
@@ -699,6 +699,264 @@ def _progress(store, done, total):
     print(f"  sqlite-vec tables recreated at dim {result['dim']}")
 
 
+def cmd_hygiene(args):
```

Ruff (0.15.20)

[warning] Too many branches (25 > 12)

(PLR0912)